content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -196,6 +196,7 @@ different Kubernetes components.
 | `BlockVolume` | `false` | Alpha | 1.9 | 1.12 |
 | `BlockVolume` | `true` | Beta | 1.13 | 1.17 |
 | `BlockVolume` | `true` | GA | 1.18 | - |
+| `CRIContainerLogRotation` | `true` | GA| 1.21 | - |
```

Comment:
Rather than just adding a new line to the second table in this page, please also move the existing lines related to `CRIContainerLogRotation` here so readers have a better idea of the whole history of this feature.

Reply:
Fixed